db/migrate/20151123010203_initial_db.rb

```diff
@@ -1,9 +1,21 @@
 class InitialDb < ActiveRecord::Migration[4.2]
   def self.up
-    create_table 'categories' do |t|
-      t.string 'name', limit: 255, null: false
-    end unless ActiveRecord::Base.connection.table_exists?('categories')
+    self.create_categories
+    self.create_comments
+    self.create_slides
+    self.create_tags
+    self.create_users
+  end
+
+  def self.create_categories
+    unless ActiveRecord::Base.connection.table_exists?('categories')
+      create_table 'categories' do |t|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->